crates/ruff_server/src/server.rs

```diff
@@ -191,6 +191,7 @@ impl Server {
                     code_action_kinds: Some(vec![
                         CodeActionKind::QUICKFIX,
                         CodeActionKind::SOURCE_ORGANIZE_IMPORTS,
+                        CodeActionKind::SOURCE_FIX_ALL,
```

Comment:
Ruff's current LSP also supports a ruff specific `fixAll` action. It's [documented here](https://github.com/astral-sh/ruff-vscode?tab=readme-ov-file#configuring-vs-code) towards the end.

```json
{
  "[python]": {
    "editor.codeActionsOnSave": {
      "source.fixAll.ruff": "explicit",
      "source.organizeImports.ruff": "explicit"
    }
  }
}
```